src/Administration/Resources/app/administration/src/module/sw-settings-shipping/page/sw-settings-shipping-detail/index.js

```diff
@@ -212,8 +212,10 @@ export default {
 
         loadCurrencies() {
             this.currenciesLoading = true;
-            this.currencyRepository.search(new Criteria(1, 500), Context.api).then((currencyResponse) => {
-                Shopware.State.commit('swShippingDetail/setCurrencies', this.sortCurrencies(currencyResponse));
+            const criteria = new Criteria(1, 500);
+            criteria.addAssociation('salesChannels');
+            this.currencyRepository.search(criteria, Context.api).then((currencyResponse) => {
+                Shopware.State.get('swShippingDetail').currencies = this.sortCurrencies(currencyResponse);
```

Comment:
Is there a reason, why you didn't use the `.commit('swShippingDetail/setCurrencies')` anymore?